resources/fixtures/Clay_Paky/Clay-Paky-Alpha-Spot-575-HPE.qxf

```diff
@@ -93,28 +93,28 @@
   <Capability Min="48" Max="55" Preset="GoboMacro" Res1="ClayPaky/gobo00020.svg">Gobo 6</Capability>
   <Capability Min="56" Max="63" Preset="GoboMacro" Res1="ClayPaky/gobo00057.svg">Gobo 7</Capability>
   <Capability Min="64" Max="71" Preset="GoboMacro" Res1="ClayPaky/gobo00052.svg">Gobo 8</Capability>
-  <Capability Min="72" Max="113">Fast to slow rotation</Capability>
-  <Capability Min="114" Max="117">Stop rotation</Capability>
-  <Capability Min="118" Max="159">Slow to fast rotation</Capability>
-  <Capability Min="160" Max="255">Gobos shake</Capability>
+  <Capability Min="72" Max="113" Preset="RotationClockwiseFastToSlow">Fast to slow rotation</Capability>
+  <Capability Min="114" Max="117" Preset="RotationStop">Stop rotation</Capability>
+  <Capability Min="118" Max="159" Preset="RotationCounterClockwiseSlowToFast">Slow to fast rotation</Capability>
+  <Capability Min="160" Max="255" Preset="GoboShakeMacro" Res1="">Gobos shake</Capability>
  </Channel>
  <Channel Name="Rotating Gobo 1 change">
   <Group Byte="0">Gobo</Group>
-  <Capability Min="0" Max="17">No gobo</Capability>
+  <Capability Min="0" Max="17" Preset="GoboMacro" Res1="Other/Open.svg">No gobo</Capability>
   <Capability Min="18" Max="35" Preset="GoboMacro" Res1="ClayPaky/gobo00067.svg">Gobo 1</Capability>
   <Capability Min="36" Max="53" Preset="GoboMacro" Res1="ClayPaky/gobo00059.svg">Gobo 2</Capability>
   <Capability Min="54" Max="71" Preset="GoboMacro" Res1="ClayPaky/gobo00068.svg">Gobo 3</Capability>
   <Capability Min="72" Max="89" Preset="GoboMacro" Res1="ClayPaky/gobo00069.svg">Gobo 4</Capability>
   <Capability Min="90" Max="107" Preset="GoboMacro" Res1="ClayPaky/gobo00070.svg">Gobo 5</Capability>
   <Capability Min="108" Max="129" Preset="GoboMacro" Res1="ClayPaky/gobo00071.svg">Gobo 6</Capability>
-  <Capability Min="130" Max="255">Gobo shake</Capability>
+  <Capability Min="130" Max="255" Preset="GoboShakeMacro" Res1="">Gobo shake</Capability>
```

Comment:
this is detailed in the manual
130-150 gobo 1
151-171 gobo 2
172-192 gobo 3
193-213 gobo 4
214-234 gobo 5
235-255 gobo 6